rclcpp/include/rclcpp/executor.hpp

```diff
@@ -813,6 +823,8 @@ class Executor
 
   rmw_guard_condition_t * interrupt_guard_condition_;
 
+  memory_strategy::MemoryStrategy::SharedPtr _memory_strategy;
```

Comment:
The member variable should have a trailing underscore and no leading underscore to be consistent with other code.